CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ ENDIF(ENABLE_ORC)
 pkg_check_modules(pkgs REQUIRED ${PKG_MODULES})
 
 INCLUDE_DIRECTORIES(
-	${CMAKE_SOURCE_DIR}/gst/nnstreamer
+	${CMAKE_SOURCE_DIR}/gst/nnstreamer/common
```

Comment:
Let's keep the common files at /gst/nnstreamer/
We do not need "common" subdirectory

Reply:
@myungjoo updated to remove common folder